### DISCUSSION TITLE
Welcome to devdocs-for-tests Discussions!

### DISCUSSION THREAD

**Discussion by dshevtsov** [magento-devdocs/devdocs-for-tests#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeff-matthews** [comment]
Pretty cool. It seems that "Discussions" are similar to "Issues".

**Reply by dshevtsov** [reply]
Yes, but with more features, like this thread for example.
